repose-aggregator/src/docs/asciidoc/recipes/docker.adoc

```diff
@@ -19,21 +13,18 @@ To run Repose in Docker, there are just two basic steps:
 .. Build the Docker image. See: Dockerfile
```

Comment:
Link "Docker Image" and "Dockerfile".

Swap those so that the order matches the order on the page.